cocotb/log.py

```diff
@@ -40,10 +40,10 @@
 
 import cocotb.ANSI as ANSI
 
-if "COCOTB_REDUCED_LOG_FMT" in os.environ:
-    _suppress = True
-else:
-    _suppress = False
+try:
+    _suppress = int(os.environ.get("COCOTB_REDUCED_LOG_FMT", "1"))
```

Comment:
```suggestion
    _suppress = int(os.getenv("COCOTB_REDUCED_LOG_FMT", default="1"))
```
The existing codebase leans toward `getenv` I think.

Reply:
No strong feelings wither way, but I would also go for `os.environ.get()` :-)

Reply:
And this is why "There should be one &mdash; and preferably only one &mdash; obvious way to do it." =)